src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -496,7 +496,51 @@ def compute_sliding_rp_violations(
 )
 
 
-def compute_synchrony_metrics(sorting_analyzer, synchrony_sizes=(2, 4, 8), unit_ids=None, **kwargs):
+def get_synchrony_counts(spikes, synchrony_sizes, all_unit_ids):
+    """Compute synchrony counts, the number of simultaneous spikes with sizes `synchrony_sizes`
+
+    Parameters
+    ----------
+    spikes : memmap
+        The spike train
+    synchrony_sizes : numpy array
```

Comment:
Is this given in seconds, samples, milliseconds?

Reply:
I've made the description more accruate (since it's actually a structured numpy array). One of the fields is "sample_index" which clarifies the unit.

Reply:
I was talking about the synchrony_sizes

Reply:
Oh, sorry. These are the number of synchronous events you want to count. So if you want to see when two or four spikes fire at the same time you use synchrony_sizes = (2,4). So it's an integer, and I think it will be clear for anyone who knows enough about the metric that their using it.

Reply:
Thanks. I was my lack of familarity is evident then. I somehow imagined that it was the windows in which an event would be count as syncronouse. Thanks for explaining.